command/repo.go

```diff
@@ -414,12 +418,91 @@ func repoView(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return err
 	}
-	_, err = api.GitHubRepo(apiClient, toView)
+	repo, err := api.GitHubRepo(apiClient, toView)
+	if err != nil {
+		return err
+	}
+
+	web, err := cmd.Flags().GetBool("web")
 	if err != nil {
 		return err
 	}
 
-	openURL := fmt.Sprintf("https://github.com/%s", ghrepo.FullName(toView))
-	fmt.Fprintf(cmd.ErrOrStderr(), "Opening %s in your browser.\n", displayURL(openURL))
-	return utils.OpenInBrowser(openURL)
+	fullName := ghrepo.FullName(toView)
+
+	openURL := fmt.Sprintf("https://github.com/%s", fullName)
+	if web {
+		fmt.Fprintf(cmd.ErrOrStderr(), "Opening %s in your browser.\n", displayURL(openURL))
+		return utils.OpenInBrowser(openURL)
+	}
+
+	repoTmpl := `
+{{.FullName}}
+{{.Description}}
+
+{{.Readme}}
+
+{{.View}}
+`
+
+	tmpl, err := template.New("repo").Parse(repoTmpl)
+	if err != nil {
+		return err
+	}
+
+	type readmeResponse struct {
+		Name    string
+		Content string
+	}
+
+	var readme readmeResponse
+
+	err = apiClient.REST("GET", fmt.Sprintf("repos/%s/readme", fullName), nil, &readme)
+	if err != nil && !strings.HasSuffix(err.Error(), "'Not Found'") {
+		return fmt.Errorf("could not get readme for repo: %w", err)
+	}
+
+	decoded, err := base64.StdEncoding.DecodeString(readme.Content)
+	if err != nil {
+		return fmt.Errorf("failed to decode readme: %w", err)
+	}
+
+	readmeContent := string(decoded)
+
+	if strings.HasSuffix(readme.Name, ".md") {
```

Comment:
In my experience, GitHub supports the following extensions as markdown:
- `.md`
- `.mdown`
- `.mkdown`
- `.markdown`

Reply:
ahh good catch